android/src/main/java/ee/forgr/audio/NativeAudio.java

```diff
@@ -84,6 +87,13 @@ public class NativeAudio extends Plugin implements AudioManager.OnAudioFocusChan
     private static final int NOTIFICATION_ID = 1001;
     private static final String CHANNEL_ID = "native_audio_channel";
 
+    // Track playOnce assets for automatic cleanup
+    private Set<String> playOnceAssets = new HashSet<>();
+
+    /**
+     * Initializes plugin runtime state by obtaining the system AudioManager, preparing the asset map,
+     * and recording the device's original audio mode without requesting audio focus.
+     */
```

Comment:
If you want to do Javadocs, please make sure to link to AudioManager like so:

```suggestion
    /**
     * Initializes plugin runtime state by obtaining the system {@link AudioManager}, preparing the asset map,
     * and recording the device's original audio mode without requesting audio focus.
     */
```

Also this is unrelated to the `playOnce` PR, so I don't understand why this change is here

Reply:
we dont use javadoc, only jsdoc pleae:)